Gravity/Gravity.Test.Integration/RSAPI_IntegrationTest.cs

```diff
@@ -339,6 +353,12 @@ public void Valid_Gravity_RelativityObject_Read_Field_Type<T>(string objectPrope
                         kCura.Relativity.Client.DTOs.Choice singleChoiceToAdd = new kCura.Relativity.Client.DTOs.Choice(singleChoiceGuid);
                         dto.Fields.Add(new FieldValue(testFieldGuid, singleChoiceToAdd));
                         break;
+                    case RdoFieldType.SingleObject:
+                        int objectToAttach =
+                            _testObjectHelper.CreateTestObjectWithGravity<GravityLevel2>(sampleData as GravityLevel2);
+                        dto.Fields.Add(new FieldValue(testFieldGuid, objectToAttach));
+                        expectedData = (sampleData as GravityLevel2).Name;
```

Comment:
If you know that something is supposed to be of a specific type, it is better to use casting than an `as` statement. Casting provides better compile-time checking and fails at the point of casting, rather than getting a null you do not intend.